src/main.rs

```diff
@@ -36,8 +39,10 @@ fn forecast(world: WorldState, depth: u8) -> (Vec<(Commit, f32)>, Duration) {
 
 
 fn oppose(in_medias_res: WorldState, depth: u8) -> (Commit, Duration) {
-    let (forecasts, thinking_time) = forecast(in_medias_res, depth);
-    let (determination, _karma) = forecasts[0];
+    let (mut forecasts, thinking_time) = forecast(in_medias_res, depth);
+    let (determination, _karma) = forecasts
+        .pop()
```

Comment:
@FoxLisk FYI, this is a bug (only affecting web client rather than console play).

![move_selection_bug](https://cloud.githubusercontent.com/assets/1076988/9724184/0dccaa0c-5588-11e5-9e20-c549b7830d19.png)

Fixed in 707db828.